packages/cxl-ui/src/components/cxl-dashboard-header.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/progress-bar';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlDashboardHeaderGlobalStyles from '../styles/global/cxl-dashboard-header-css.js';
+import cxlDashboardHeaderStyles from '../styles/cxl-dashboard-header-css.js';
+
+@customElement('cxl-dashboard-header')
+export class CXLDashboardHeaderElement extends LitElement {
+  static get styles() {
+    return [cxlDashboardHeaderStyles];
+  }
+
+  @property({ type: String }) name = '';
+
+  @property({ type: String }) notificationCount = '6';
+
+  @property({ type: String }) updateTitle = 'What\'s new in CXL';
+
+  @property({ type: String }) lastCourseTitle = 'Advanced experimentation analysis';
+
+  @property({ type: String }) allCourses = 'Browse all courses';
+
+  @property({ type: String }) fastSkillsCourses = 'Browse fast skills';
+
+  @property({ type: String }) createRoadmap = 'Create your personal learning roadmap';
+
+  @property({ type: Boolean }) hasRoadmap = false;
+
+  renderLastCourse() {
+    return this.hasRoadmap
+    ? html`
+      <div class="last-course">
+        <vaadin-button>
+          <div>
+            <span class="subtitle">Continue where you left off</span>
+            <span class="title">${this.lastCourseTitle}</span>
+          </div>
+          <vaadin-icon icon="lumo:arrow-right"></vaadin-icon>
+        </vaadin-button>
+        <div class="progress">
+          <span class="progress-title">Completed 2 lessons of 6 lessons</span>
+          <vaadin-progress-bar value="0.334">Completed 2 lessons of 6 lessons</vaadin-progress-bar>
+        </div>
+      </div>`
+    : nothing;
+  }
+
+  renderStats() {
+    return this.hasRoadmap
+    ? html`
+      <div class="stats">
+        <slot name="stats-desktop"></slot>
+        <vaadin-details theme="cxl-dashboard-header reverse" class="vaadin-details">
+          <div slot="summary">Your roadmap</div>
+          <slot name="stats-mobile"></slot>
+        </vaadin-details>
+      </div>`
+    : nothing;
+  }
+
+  render() {
+    return html`
+      <div class="container">
+        <header>
+          <vaadin-button class="updates">
+            <vaadin-icon class="icon-bell" count="${this.notificationCount}" icon="lumo:bell"></vaadin-icon>
+            ${this.updateTitle}
+            <vaadin-icon icon="lumo:angle-right"></vaadin-icon>
+          </vaadin-button>
+          <div>
+            <span class="subtitle">My dashboard</span>
+            <h1 class="title">Hello ${this.name}!</h1>
```

Comment:
I suggest making these properties with default values:
* `title` -> 'Hello'
* `subtitle` -> 'My dashboard'

We need more flexibility and changing the header should not require editing a component.